src/run/runner/shared/fifo.rs

```diff
@@ -129,7 +127,7 @@ impl RunnerFifo {
         &mut self,
         mut health_check: impl AsyncFnMut() -> anyhow::Result<bool>,
         mut handle_cmd: impl AsyncFnMut(&FifoCommand) -> anyhow::Result<FifoCommand>,
-    ) -> anyhow::Result<FifoBenchmarkData> {
+    ) -> anyhow::Result<(MarkerResult, FifoBenchmarkData)> {
```

Comment:
I think some of the naming was not properly fixed up here, FYI.

Don't necessarily bother redoing everything tho